src/flick/friend/serializers.py

```diff
@@ -28,3 +28,42 @@ class Meta:
         model = User
         fields = ("id", User.USERNAME_FIELD, "first_name", "last_name", "profile")
         read_only_fields = fields
+
+
+class FriendRequestSerializer(serializers.ModelSerializer):
+    """
+    Friend Request Serializer
+    """
+
+    to_user = FriendUserSerializer(read_only=True)
```

Comment:
what's to_user? is this a serialized object friend? you should name it "friend" then?

Reply:
we are using the friend request modal from django-friendship and it only has from_user and to_user. For an outgoing request, we think it is not necessary to include the from_user(the user themselves) so we only put to_user there. Do you want us to override the name 'to_user' to be 'friend'?